### ISSUE TITLE
https://freshandfit.locals.com/

### ISSUE THREAD

**Issue by hasnogaems** [yt-dlp/yt-dlp#7578]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Example URLs

-Single Video https://freshandfit.locals.com/post/4280045/stream-with-the-ladies

### Provide a description that is worded well enough to be understood

Trying to download this video. It is opened only after registration. The issue is it is chunked into .ts files and the only m3u8 files I been able to get only produce short chunk.
How can I actually download whole video?

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU https://freshandfit.locals.com/api/v1/posts/content/hls/v_be4c7342-4f2d-49f2-a47b-dc94b8c5712f/source.m3u8
[debug] Command-line config: ['-vU', 'https://freshandfit.locals.com/api/v1/posts/content/hls/v_be4c7342-4f2d-49f2-a47b-dc94b8c5712f/source.m3u8']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version nightly@2023.07.11.010054 [2af4eeb77] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-90610-g2accdd3871, ffprobe N-90610-g2accdd3871
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1856 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-nightly-builds/releases/latest
Available version: nightly@2023.07.11.010054, Current version: nightly@2023.07.11.010054
Current Build Hash: fd6e55986ae0904f69c03c9e31b85539908c6c9ba1603400eee5124f5c3d30a2
yt-dlp is up to date (nightly@2023.07.11.010054)
[generic] Extracting URL: https://freshandfit.locals.com/api/v1/posts/content/hls/v_be4c7342-4f2d-49f2-a47b-dc94b8c5712f/source.m3u8
[generic] source: Downloading webpage
[debug] Identified a direct video link
[generic] source: Downloading m3u8 information
[generic] source: Checking m3u8 live status
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] source: Downloading 1 format(s): 4000
[debug] Invoking hlsnative downloader on "https://freshandfit.locals.com/api/v1/posts/content/hls/playlist/v_be4c7342-4f2d-49f2-a47b-dc94b8c5712f/1080.m3u8"
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 1
[download] Destination: source [source].mp4
[debug] File locking is not supported. Proceeding without locking
[download] 100% of    8.70MiB in 00:00:00 at 8.73MiB/s
[debug] ffprobe command line: ffprobe -hide_banner -show_format -show_streams -print_format json "file:source [source].mp4"
[debug] ffmpeg command line: ffprobe -show_streams "file:source [source].mp4"
[FixupM3u8] Fixing MPEG-TS in MP4 container of "source [source].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:source [source].mp4" -map 0 -dn -ignore_unknown -c copy -f mp4 "-bsf:a" aac_adtstoasc -movflags "+faststart" "file:source [source].temp.mp4"
```

**Comment by bashonly** [comment]
Duplicate of #1827
